### PR TITLE
archinstall, blkid, btrfs-*: add pt_BR translation

### PR DESCRIPTION
**Pull request by isaacvicente**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by isaacvicente** [comment]
Apparently the CI test failed because there's a blank line at the end of each file. Can a maintainer suggest the necessary changes?

**Comment by isaacvicente** [comment]
> Apparently the CI test failed because there's a blank line at the end of each file. Can a maintainer suggest the necessary changes?

Nevermind. I made the changes myself :)

File: pages.pt_BR/linux/archinstall.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# archinstall
+
+> Instalador do Arch Linux guiado com uma torção.
+> Mais informações: <https://archinstall.readthedocs.io>.
+
+- Inicie o instalador interativo:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/archinstall.md:6]:
```suggestion
- Inicia o instalador interativo:
```

File: pages.pt_BR/linux/archinstall.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+`archinstall`
+
+- Inicie um instalador predefinido:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/archinstall.md:10]:
```suggestion
- Inicia um instalador predefinido:
```

File: pages.pt_BR/linux/archinstall.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+`archinstall {{mínimo|autônomo}}`
```
Comment (navarroaxel) [inline pages.pt_BR/linux/archinstall.md:12]: This arguments can't be translated
```suggestion
`archinstall {{minimal|unattended}}`
```

File: pages.pt_BR/linux/btrfs-check.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Verifica e repara um sistema de arquivos btrfs (perigoso):
+
+`sudo btrfs check --repair {{caminho/para/partição}}`
+
+- Mostra o andamento da verificação:
+
+`sudo btrfs check --progress {{caminho/para/partição}}`
+
+- Verifique a soma de verificação de cada bloco de dados (se o sistema de arquivos estiver bom):
```
Comment (navarroaxel) [inline pages.pt_BR/linux/btrfs-check.md:18]:
```suggestion
- Verifica a soma de verificação de cada bloco de dados (se o sistema de arquivos estiver bom):
```

File: pages.pt_BR/linux/btrfs-device.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Adiciona um ou mais dispositivos a um sistema de arquivos btrfs:
+
+`sudo btrfs device add {{caminho/para/bloco_do_dispositivo1}} [{{caminho/para/bloco_do_dispositivo2}}] {{caminho/para/sistema_de_arquivos_btrfs}}`
+
+- Remova um dispositivo de um sistema de arquivos btrfs:
+
+`sudo btrfs device remove {{caminho/para/dispositivo|id_do_dispositivo}} [{{...}}]`
+
+- Exiba estatísticas de erro:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/btrfs-device.md:14]:
```suggestion
- Exibe estatísticas de erro:
```

File: pages.pt_BR/linux/btrfs-device.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# btrfs device
+
+> Gerencia dispositivos em um sistema de arquivos btrfs.
+> Mais informações: <https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-device>.
+
+- Adiciona um ou mais dispositivos a um sistema de arquivos btrfs:
+
+`sudo btrfs device add {{caminho/para/bloco_do_dispositivo1}} [{{caminho/para/bloco_do_dispositivo2}}] {{caminho/para/sistema_de_arquivos_btrfs}}`
+
+- Remova um dispositivo de um sistema de arquivos btrfs:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/btrfs-device.md:10]:
```suggestion
- Remove um dispositivo de um sistema de arquivos btrfs:
```

File: pages.pt_BR/linux/btrfs-device.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+- Exiba estatísticas de erro:
+
+`sudo btrfs device stats {{caminho/para/sistema_de_arquivos_btrfs}}`
+
+- Examina todos os discos e informa ao kernel todos os sistemas de arquivos btrfs detectados:
+
+`sudo btrfs device scan --all-devices`
+
+- Exiba estatísticas detalhadas de alocação por disco:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/btrfs-device.md:22]:
```suggestion
- Exibe estatísticas detalhadas de alocação por disco:
```

**Comment by navarroaxel** [comment]
Remember that you can use this GitHub feature <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request> to apply all the suggestions in 1 commit.

File: pages.pt_BR/linux/btrfs-rescue.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+- Recupera um superblock corrompido das cópias corretas (recupere a raiz da árvore do sistema de arquivos):
+
+`sudo btrfs rescue super-recover {{caminho/para/partição}}`
+
+- Recupera-se de uma transação interrompida (corrige problemas de repetição de log):
+
+`sudo btrfs rescue zero-log {{caminho/para/partição}}`
+
+- Crie um dispositivo de controle `/dev/btrfs-control` quando o `mknod` não estiver instalado:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/btrfs-rescue.md:22]:
```suggestion
- Cria um dispositivo de controle `/dev/btrfs-control` quando o `mknod` não estiver instalado:
```

**Comment by isaacvicente** [comment]
> Remember that you can use this GitHub feature https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request to apply all the suggestions in 1 commit.

Oh, thank you :) I didn't know about this.

File: pages.pt_BR/linux/btrfs-property.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+- Obtém a propriedade `label` (etiqueta) para o sistema de arquivos ou dispositivo btrfs fornecido:
+
+`sudo btrfs property get {{caminho/para/sistema_de_arquivos_btrfs}} label`
+
+- Obtém todas as propriedades específicas do tipo de objeto para o sistema de arquivos ou dispositivo btrfs fornecido:
+
+`sudo btrfs property get -t {{subvol|filesystem|inode|device}} {{caminho/para/sistema_de_arquivos_btrfs}}`
+
+- Defina a propriedade de `compression` (compactação) para um determinado inode btrfs (um arquivo ou diretório):
```
Comment (navarroaxel) [inline pages.pt_BR/linux/btrfs-property.md:22]:
```suggestion
- Define a propriedade de `compression` (compactação) para um determinado inode btrfs (um arquivo ou diretório):
```

**Review by navarroaxel — APPROVED** [review]

**Comment by isaacvicente** [comment]
I tried to commit all changes at once, but I couldn't... (I read the link you gave me, btw)

**Comment by kbdharun** [comment]
> I tried to commit all changes at once, but I couldn't... (I read the link you gave me, btw)

You can go to files changes tab and add all suggestion to batch and create a single commit in it.

**Comment by isaacvicente** [comment]
> > I tried to commit all changes at once, but I couldn't... (I read the link you gave me, btw)
> 
> You can go to the files changed tab and add all suggestions to the batch and create a single commit in it.

That's why I couldn't commit all the changes... I was at the conversation tab. Thanks man!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**